api/nodes/serializers.py

```diff
@@ -890,7 +890,9 @@ def create(self, validated_data):
 class ForwardNodeAddonSettingsSerializer(NodeAddonSettingsSerializerBase):
```

Comment:
oh my goodness.  We have two separate v2 views/serializers which handle updating the forward addon.  
 
PATCH http://localhost:8000/v2/nodes/9bknu/addons/forward/ implemented by Matt in 2016 https://github.com/CenterForOpenScience/osf.io/pull/6122

PATCH http://localhost:8000/v2/nodes/9bknu/settings/.  Erin and I implemented this last year as part of the NodeSettings endpoint https://github.com/CenterForOpenScience/osf.io/pull/8536.  

John's work adds the spam check in Matt's v2 endpoint, but not in ours - NodeSettingsUpdateSerializer.  **At minimum, both v2 serializers should handle spam checking on the forward addon, but it's my opinion we should deprecate one of them in a follow-up PR, after Emberization occurs.** 

Matt's: added first, current legacy page uses this v1 view, appears to be documented

Dawn's/Erin's: appears to be documented, if I remember right, this was planned as part of the emberized work? Does Emberized page plan to use to the NodeSettingsSerializer to update the forward addon? (Can you weigh in tomorrow @brianjgeiger ?)